tests/testSampleStructures.js

```
@@ -159,85 +63,40 @@
+		req.on("data", chunk => {
+			dataChunks.push(chunk);
+		});
+		req.on("end", () => {
+			let fileBuffer = Buffer.concat(dataChunks);
+			let fileName = req.headers["content-location"];
+			let filePath = path.join(packUploadDir, fileName);
+			
+			fs.writeFile(filePath, fileBuffer, async err => {
+				if(browserEngine == "chrome") {
+					try {
+						let previewImage = await page.waitForSelector("#previewCont > canvas:last-child");
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'previewImage' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)